runtime/onert/odc/MinMaxReader.cc

```diff
@@ -147,7 +160,11 @@ MinMaxVectors MinMaxReader::readInput(uint32_t model_idx, uint32_t subg_idx,
     readMMFile(&num_input, sizeof(uint32_t), 1, file, "Cannot read num of inputs");
 
     // Skip operation minmax data
-    std::fseek(file, data_size * num_op, SEEK_CUR);
+    if (std::fseek(file, static_cast<int64_t>(data_size * num_op), SEEK_CUR))
```

Comment:
Maybe..? 

```suggestion
    if (std::fseek(file, static_cast<int64_t>(data_size * num_op), SEEK_CUR) != 0)
```

https://en.cppreference.com/w/cpp/io/c/fseek said `fseek` returns nonzero if failed. 
So I thought that the logic might be like this : 
```cpp 
if (fseek failed) {
	// throw runtime error 
}
```